### ISSUE TITLE
Keyboard focus indicator is visible only on second tab when focus moves on table headers present in the page.

### ISSUE THREAD

**Issue by niragvt** [mbrn/material-table#2723]
# Guidelines

- #### Please include a demo of the issue/behavior/question you have

- #### Please try to be as detailed as possible

- #### You may fork one of the following starter templates if you would like:

  - #### [CodeSandbox](https://codesandbox.io/s/material-table-starter-template-xnfpo)
  - #### [StackBlitz](https://stackblitz.com/edit/material-table-starter-template)

**Describe the bug**
We are using material-table and got an accessibility bug related to focus on the table columns header when using keyboard tab. The column header is in focus only on tabbing twice 

**To Reproduce**

Step 1: Go to any page which has material-table control.
Step 2: Verify whether the Keyboard focus indicator is not visible when focus moves on table headers present in the page or not.

**Actual Result**
1. Keyboard focus indicator is not visible when focus moves on table headers present in the page.
Observation: On second time tab focus, focus indicator is visible for table headers.
2. Keyboard focus indicator is not visible when focus moves on ''Rows per page'' combo box present in the page.

**Expected behavior**
Keyboard focus indicator should be visible clearly when focus moves on table headers for the first time and for ''Rows per page'' combo box present in the page.
Ensure the keyboard focus indicator should be visible for lists present in combo box.

**Screenshots**

![image](https://user-images.githubusercontent.com/48108598/103664937-abd34280-4f27-11eb-9df9-f21d522f7a98.png)

**Desktop (please complete the following information):**

- OS: Windows 
- Browser Chrome and edge

**Additional context**
